bot.py

```diff
@@ -85,6 +84,17 @@ async def off_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
         await update.message.reply_text("You don't have an active session. Use /on to start the timer.")
 
+async def abort_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_name = update.effective_user.username
+    if user_name in user_sessions:
+        if user_name == "Unknown":
```

Comment:
Need to use this user_name = USER_NAME_MAPPING.get(user_name, "Unknown")
Also please remove str() from line 75

Reply:
Also move check for unknown user to on_command. Dont need check again here.

Reply:
Resolved! 

![Screenshot 2023-07-19 at 3 34 07 PM](https://github.com/samtjong23/airconTeleBot/assets/65160072/84d27049-c1b0-47a0-b870-ea29d5b31894)